easybuild/easyconfigs/n/NAGfor/NAGfor-6.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Binary'
+
+name = 'NAGfor'
+version = '6.2'
+
+homepage = 'http://www.nag.co.uk'
+description = """The checking compiler for improved code portability and detailed error reporting."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+nagname = 'npl%(version_major)sa%(version_major)s%(version_minor)sna_amd64'
+
+source_urls = ['https://www.nag.co.uk/downloads/impl']
+sources = [nagname + '.tgz']
+checksums = ['4fce371e799d321c7842f867b8a98e64d54cc806da43d801c77e3645f295c1d4']
+
+preinstallopts = 'tar -xvf %s.tgz && ' % nagname
+preinstallopts += 'cd NAG_Fortran-amd64 && '
+preinstallopts += 'INSTALL_TO_BINDIR=%(installdir)s/bin '
+preinstallopts += 'INSTALL_TO_LIBDIR=%(installdir)s/lib '
+preinstallopts += 'INSTALL_TO_CATMANDIR=%(installdir)s/man '
+preinstallopts += 'INSTALL_TO_MANDIR=%(installdir)s/man '
+install_cmd = './INSTALLU.sh'
+
+modextravars = {
+    'NAG_KUSARI_FILE': HOME + '/licenses/license.dat',
```

Comment:
What if both `NAG` and `NAGfor` modules are loaded, the value for `$NAG_KUSARI_FILE` would be overwritten by the module that is loaded last?

Reply:
For us, the same licence file covers all our NAG libraries and compilers.  If they're different then these will have to be edited before installing.  We use a licence server so we change to `<machine_name:>`.  Is there a way this could be passed on the command line at installation time?